mmengine/structures/base_data_element.py

```diff
@@ -5,6 +5,7 @@
 
 import numpy as np
 import torch
+from torch.utils.data._utils.pin_memory import pin_memory
```

Comment:
it is not safe to call a private method

Reply:
If you can't call it, we can only rewrite it?